burn-import/onnx-tests/tests/onnx_tests.rs

```diff
@@ -333,6 +334,15 @@ mod tests {
         assert_eq!(output.to_data(), expected);
     }
 
+
+    #[test]
+    fn shape() {
+        let model: shape::Model<Backend> = shape::Model::new();
+        let output = model.forward();
+        assert_eq!(output, (1,1,1,4));
+    }
```

Comment:
Shapes in burn are not defined by tuples, but by the `Shape<D>` struct. I think to properly support shapes with `burn-import` we need to output the correct shape type, but also implement `Tensor<B, D, Int>` from shape, since shapes are sometimes used as int tensor by the onnx graph. @antimora

Reply:
@antimora can you provide your view? I am not sure I understood the recommandation

Reply:
The tuple result is erroneous because the ONNX returns four outputs because of constants. See my comment below. 

tensor.shape() would return `Shape<D>` already so we should use this type. This result can be re-fed to `reshape` operation, which is a good test. Please see my comment below.